envs/tensorrt-env.yaml

```diff
@@ -1,8 +1,11 @@
 packages:
 {% set py_ver = python | float %}
-{% if build_type == 'cuda' and py_ver < 3.8 %}
+{% if build_type == 'cuda' %}
```

Comment:
Approved this if cpu builds work fine with this.

Reply:
Yes, cpu builds are fine.

Reply:
We might still need to add libc_wrappers to other environment files. Are there any environment files that include CuDNN, but not TensorRT?

Reply:
no change in cudnn repo for libc_wrappers.

Reply:
Currently there is no other env file that uses cudnn (and not TRT), but I suppose there could be. Or someone could create a custom one, in which case they'd have to know to put in the wrappers.